holland/legacy/config.py

```diff
@@ -25,8 +25,14 @@ def extract_params(section):
 def get_config(**kwargs):
     global config
     global spec
+
+    holland_conf = '/etc/holland/holland.conf'
+    currplatform = sys.platform
```

Comment:
This `holland/legacy/config.py` file is dead code and should be deleted.  Sloppy practices from a long time ago that never got cleaned up. :(